### PR TITLE
Style: Add period before </param> and </exception> doc tags

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Add period before </param> and </exception> doc tags.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: src/System.Management.Automation/engine/parser/ast.cs [hunk 1]
```diff
@@ -8118,7 +8118,7 @@ public override int GetHashCode()
         /// <summary>
         /// Check if the type names a <see cref="System.Type"/>, false otherwise.
         /// </summary>
-        /// <param name="type">The given <see cref="System.Type"/></param>
+        /// <param name="type">The given <see cref="System.Type"/>.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/parser/ast.cs]: Leave as empty

File: src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/typeDataXmlLoader.cs [hunk 2]
```diff
@@ -274,11 +274,11 @@ private static class XMLStringValues
...
         /// </summary>
-        /// <param name="typeDefinition">the ExtendedTypeDefinition instance to load formatting data from</param>
-        /// <param name="db">database instance to load the formatting data into</param>
-        /// <param name="expressionFactory">expression factory to validate the script block</param>
-        /// <param name="isBuiltInFormatData">do we implicitly trust the script blocks (so they should run in full langauge mode)?</param>
-        /// <param name="isForHelp">true when the view is for help output</param>
+        /// <param name="typeDefinition">the ExtendedTypeDefinition instance to load formatting data from.</param>
+        /// <param name="db">database instance to load the formatting data into.</param>
+        /// <param name="expressionFactory">expression factory to validate the script block.</param>
+        /// <param name="isBuiltInFormatData">do we implicitly trust the script blocks (so they should run in full langauge mode)?.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/typeDataXmlLoader.cs]: Already has punctionation

File: src/System.Management.Automation/engine/lang/parserutils.cs [hunk 3]
```diff
@@ -1143,12 +1143,12 @@ internal static object LikeOperator(ExecutionContext context, IScriptExtent erro
...
-        /// <param name="context">The execution context to use</param>
+        /// <param name="context">The execution context to use.</param>
         /// <param name="errorPosition">The position to use for error reporting.</param>
-        /// <param name="lval">left operand</param>
-        /// <param name="rval">right operand</param>
-        /// <param name="ignoreCase">ignore case?</param>
-        /// <param name="notMatch">true for -notmatch, false for -match</param>
+        /// <param name="lval">left operand.</param>
+        /// <param name="rval">right operand.</param>
+        /// <param name="ignoreCase">ignore case?.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/lang/parserutils.cs]: Already has punctuation

File: src/System.Management.Automation/help/DefaultCommandHelpObjectBuilder.cs [hunk 4]
```diff
@@ -402,13 +402,13 @@ internal static void AddParametersProperties(PSObject obj, Dictionary<string, Pa
...
-        /// <param name="name">parameter name</param>
-        /// <param name="aliases">parameter aliases</param>
-        /// <param name="dynamic">is dynamic parameter?</param>
-        /// <param name="type">parameter type</param>
-        /// <param name="attributes">parameter attributes</param>
-        /// <param name="parameterSetName">Name of the parameter set for which the syntax is generated</param>
+        /// <param name="obj">HelpInfo object.</param>
+        /// <param name="name">parameter name.</param>
+        /// <param name="aliases">parameter aliases.</param>
+        /// <param name="dynamic">is dynamic parameter?.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/help/DefaultCommandHelpObjectBuilder.cs]: Already has punctuation

File: src/System.Management.Automation/help/UpdatableHelpCommandBase.cs [hunk 5]
```diff
@@ -716,9 +716,9 @@ internal bool CheckOncePerDayPerModule(string moduleName, string path, string fi
         /// <summary>
         /// Resolves a given path to a list of directories
         /// </summary>
-        /// <param name="path">path to resolve</param>
-        /// <param name="recurse">resolve recursively?</param>
-        /// <param name="isLiteralPath">Treat the path / start path as a literal path?</param>///
+        /// <param name="path">path to resolve.</param>
+        /// <param name="recurse">resolve recursively?.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/help/UpdatableHelpCommandBase.cs]: Already has punctuation

File: src/System.Management.Automation/help/UpdatableHelpCommandBase.cs [hunk 6]
```diff
@@ -716,9 +716,9 @@ internal bool CheckOncePerDayPerModule(string moduleName, string path, string fi
...
+        /// <param name="isLiteralPath">Treat the path / start path as a literal path?.</param>///
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/help/UpdatableHelpCommandBase.cs]: Already has punctuation

File: src/System.Management.Automation/help/UpdatableHelpSystem.cs [hunk 7]
```diff
@@ -611,11 +611,11 @@ private string ResolveUri(string baseUri, bool verbose)
...
-        /// <param name="xml">input xml</param>
-        /// <param name="ns">schema namespace</param>
-        /// <param name="schema">xml schema</param>
-        /// <param name="handler">validation event handler</param>
-        /// <param name="helpInfo">HelpInfo or HelpContent?</param>
+        /// <param name="xml">input xml.</param>
+        /// <param name="ns">schema namespace.</param>
+        /// <param name="schema">xml schema.</param>
+        /// <param name="handler">validation event handler.</param>
+        /// <param name="helpInfo">HelpInfo or HelpContent?.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/help/UpdatableHelpSystem.cs]: Already has punctuation

File: src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs [hunk 8]
```diff
@@ -5163,9 +5163,9 @@ private static class PInvokeDllNames
         /// <summary>
         /// Convert a Local Identifier to a Locale name
         /// </summary>
-        /// <param name="localeID">The Locale ID (LCID) to be converted</param>
-        /// <param name="localeName">Destination of the Locale name</param>
-        /// <param name="localeNameSize">Capacity of <paramref name="localeName"/></param>
+        /// <param name="localeID">The Locale ID (LCID) to be converted.</param>
+        /// <param name="localeName">Destination of the Locale name.</param>
+        /// <param name="localeNameSize">Capacity of <paramref name="localeName"/>.</param>
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs]: Leave blank

File: src/System.Management.Automation/engine/LanguagePrimitives.cs [hunk 9]
```diff
@@ -1392,9 +1392,9 @@ internal static bool IsBoolOrSwitchParameterType(Type type)
         /// The property typically won't need conversion, but it could.  The value is more likely in
         /// need of conversion.
         /// </summary>
-        /// <param name="dictionary">The dictionary that potentially implement <see cref="IDictionary&lt;TKey,TValue&gt;"/></param>
-        /// <param name="key">The object representing the key</param>
-        /// <param name="value">The value to assign</param>
+        /// <param name="dictionary">The dictionary that potentially implement <see cref="IDictionary&lt;TKey,TValue&gt;"/>.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/LanguagePrimitives.cs]: Leave blank

File: src/System.Management.Automation/engine/PSVersionInfo.cs [hunk 10]
```diff
@@ -642,8 +642,8 @@ public static SemanticVersion Parse(string version)
         /// Parse <paramref name="version"/> and return true if it is a valid <see cref="SemanticVersion"/>, otherwise return false.
         /// No exceptions are raised.
         /// </summary>
-        /// <param name="version">The string to parse</param>
-        /// <param name="result">The return value when the string is a valid <see cref="SemanticVersion"/></param>
+        /// <param name="version">The string to parse.</param>
+        /// <param name="result">The return value when the string is a valid <see cref="SemanticVersion"/>.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/PSVersionInfo.cs]: Leave blank

File: src/System.Management.Automation/engine/hostifaces/HostUtilities.cs [hunk 11]
```diff
@@ -227,8 +227,8 @@ private static string GetAllUsersFolderPath(string shellId)
         /// <summary>
         /// Gets the first <paramref name="maxLines"/> lines of <paramref name="source"/>.
         /// </summary>
-        /// <param name="source">string we want to limit the number of lines</param>
-        /// <param name="maxLines">maximum number of lines to be returned</param>
+        /// <param name="source">string we want to limit the number of lines.</param>
+        /// <param name="maxLines">maximum number of lines to be returned.</param>
         /// <returns>The first lines of <paramref name="source"/>.</returns>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/hostifaces/HostUtilities.cs]: Should be blank

File: src/System.Management.Automation/engine/hostifaces/pipelinebase.cs [hunk 12]
```diff
@@ -25,9 +25,9 @@ internal abstract class PipelineBase : Pipeline
         /// Create a pipeline initialized with a command string
         /// </summary>
         /// <param name="runspace">The associated Runspace/>.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/hostifaces/pipelinebase.cs]: Should be blank

File: src/System.Management.Automation/engine/parser/ast.cs [hunk 13]
```diff
@@ -2833,7 +2833,7 @@ public UsingStatementAst(IScriptExtent extent, HashtableAst moduleSpecification)
         /// </summary>
         /// <param name="extent">The extent of the using statement including the using keyword.</param>
         /// <param name="aliasName">The name of the alias.</param>
-        /// <param name="moduleSpecification">The module being aliased. Hashtable that describes <see cref="Microsoft.PowerShell.Commands.ModuleSpecification"/></param>
+        /// <param name="moduleSpecification">The module being aliased. Hashtable that describes <see cref="Microsoft.PowerShell.Commands.ModuleSpecification"/>.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/parser/ast.cs]: Leave blank

File: src/System.Management.Automation/engine/parser/ast.cs [hunk 14]
```diff
@@ -5956,7 +5956,7 @@ public class MergingRedirectionAst : RedirectionAst
         /// </summary>
         /// <param name="extent">The extent of the redirection.</param>
         /// <param name="from">The stream to read from.</param>
-        /// <param name="to">The stream to write to - must always be <see cref="RedirectionStream.Output"/></param>
+        /// <param name="to">The stream to write to - must always be <see cref="RedirectionStream.Output"/>.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/parser/ast.cs]: Leave blank

File: src/System.Management.Automation/engine/serialization.cs [hunk 15]
```diff
@@ -5895,7 +5895,7 @@ public PSPrimitiveDictionary()
         /// Initializes a new instance of the <see cref="PSPrimitiveDictionary"/> class with contents
         /// copied from the <paramref name="other"/> hashtable.
         /// </summary>
-        /// <param name="other">hashtable to copy into the new instance of <see cref="PSPrimitiveDictionary"/></param>
+        /// <param name="other">hashtable to copy into the new instance of <see cref="PSPrimitiveDictionary"/>.</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/serialization.cs]: Leave blank

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/engine/MshObjectTypeDescriptor.cs [hunk 16]
```diff
@@ -116,7 +116,7 @@ internal PSObjectPropertyDescriptor(string propertyName, Type propertyType, bool
         /// This method has no effect for <see cref="PSObjectPropertyDescriptor"/>.
         /// CanResetValue returns false.
         /// </summary>
-        /// <param name="component">This parameter is ignored for <see cref="PSObjectPropertyDescriptor"/></param>
+        /// <param name="component">This parameter is ignored for <see cref="PSObjectPropertyDescriptor"/>.</param>
```
[the remark this answers is not in the document]
Comment (iSazonov) [reply src/System.Management.Automation/engine/MshObjectTypeDescriptor.cs]: Fixed by pattern.

File: src/System.Management.Automation/engine/lang/parserutils.cs [hunk 17]
```diff
@@ -1279,12 +1279,12 @@ internal static object MatchOperator(ExecutionContext context, IScriptExtent err
...
-        /// <param name="context">The execution context to use</param>
+        /// <param name="context">The execution context to use.</param>
         /// <param name="errorPosition">The position to use for error reporting.</param>
-        /// <param name="left">left operand</param>
-        /// <param name="right">right operand</param>
-        /// <param name="ignoreCase">ignore case?</param>
-        /// <param name="contains">true for -contains, false for -notcontains</param>
+        /// <param name="left">left operand.</param>
+        /// <param name="right">right operand.</param>
+        /// <param name="ignoreCase">ignore case?.</param>
```
[the remark this answers is not in the document]
Comment (iSazonov) [reply src/System.Management.Automation/engine/lang/parserutils.cs]: Fixed by pattern.

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**